### DISCUSSION TITLE
jsx and client side script

### DISCUSSION THREAD

**Discussion by roccho-dev** [honojs/hono#1679] [Q&A]
Hono provides this feature?
Now I use `dangerouslySetInnerHTML` to execute script on browser.
But I want to know other option if Hono provides default option for scripting on browser.
Welcome also better idea.

Thanks.

```javascript

export const ScriptElt = () => (
  <>
    <script
      dangerouslySetInnerHTML={{
        __html: `
      // script here ...
```

**Comment by roccho-dev** [comment]
https://hono.dev/helpers/html

I'll try this at first.

**Comment by janat08** [comment]
I'd rather script tags could be used in jsx by default.
